zulip_bots/zulip_bots/bots/connect_four/connect_four.py

```diff
@@ -0,0 +1,383 @@
+# @TODO: strip trailing whitespace from commands
+# @TODO: check that user doesn't play against self, and that he cannot run both commands for both players
+# @TODO: place bot owner name in config file, allow bot owner to run special commands
+
+import re
+from copy import deepcopy
+
+# Get the path to the bot directory
+# To acquire the ConnectFourModel
+import sys
+import os
+
+# Append correct path if running bot from its own directory
+sys.path.append(os.getcwd())
+
+# Append correct path if running bot tests
+sys.path.append('zulip_bots/zulip_bots/bots/connect_four')
+
+from connect_four_controller import ConnectFourModel
+
```

Comment:
@sColin16: Could you please explain why you need to use `sys.path.append` here? I'm asking because ideally we shouldn't have to and I just want to make sure I'm not missing something here. :)